Apromore-Core-Components/Apromore-Manager/src/main/java/org/apromore/service/impl/UserMetadataServiceImpl.java

```diff
@@ -281,65 +280,94 @@ public void deleteUserMetadata(Integer usermetadataId, String username) throws U
     }
 
     @Override
-    public Set<Usermetadata> getUserMetadata(String username, List<Integer> logIds,
-                                             UserMetadataTypeEnum userMetadataTypeEnum) throws UserNotFoundException {
-
-        if (null == logIds || logIds.size() == 0) {
-            return null;
-        }
+    public Set<Usermetadata> getUserMetadataByUser(String username, UserMetadataTypeEnum userMetadataTypeEnum) throws UserNotFoundException {
 
         User user = userSrv.findUserByLogin(username);
         assert user != null;
 
```

Comment:
This line is redundant because findUserByLogin will throw a UserNotFoundException: assert user != null;